### PR TITLE
refresh 2.2. do not merge

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: about/architecture.adoc [hunk 1]
```diff
@@ -63,7 +63,7 @@ For more information about Governance and risk, see the link:../security/securit
 
 The _Observability_ component collects and reports the status and health of the {ocp-short} version 4.x, or later, managed clusters to the hub cluster. You can create custom alerts to inform you of problems with your fleet of managed clusters. Because it requires configured persistent storage, observability must be enabled after the {product-title-short} installation.  
 
-For more information about Observability, see link:../observability/observe_environments_intro.adoc#observing-environments-intro[Observing environments introduction].
+For more information about Observability, see link:../observing_environments/observe_environments_intro.adoc#observing-environments-intro[Observing environments introduction].
```
Comment (swopebe) [inline about/architecture.adoc:66]: Observability is the folder name

Comment (swopebe) [reply about/architecture.adoc:66]: @cadawson concerned this undoes the actual name of the folder?

Comment (cadawson) [reply about/architecture.adoc:66]: We understand that the folder names cause the links to be broken in github. Tested them in stage of the Portal, and they worked. I would like to get the errata out today. I am going to push to prod and we will wait to push live until you let us know that we got the right content in.

**Review by cadawson — APPROVED** [review]
Link issues, but tested and work. Thanks!

**Pull request merged**